seirmo/apps/_simulation_app.py

```diff
@@ -42,10 +43,14 @@ def _set_layout(self):
         self.app.layout = dbc.Container([
             html.H1("SEIR model"),
             dbc.Row([
-                    dbc.Col([dcc.Graph(
+                    dbc.Col([dbc.Row([dcc.Graph(
                             figure=self._fig_plot._fig, id='fig')],
```

Comment:
I'd suggest to call the id fig1 (makes it more consistent with fig2)

Reply:
Yes, I agree. But it will be changed anyway when we have subplots?